docs/getting-started/edfi-exchange/technology/student-survey-dashboard-for-teachers.md

```diff
@@ -0,0 +1,38 @@
+# Student Survey Dashboard for Teachers
+
+![Ed-Fi Logo](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/image.png)
+
+## Description
+
+Project Buzz is a mobile tool that puts roster, contact information, and student survey results into the hands of teachers.
+
+The tool provides an educator with three views into their student’s data:
+
+1. **Rostering.** This shows teachers all their students along with contact information like school email, primary guardian contact information, and contact notes, and can be filtered by class or student name.
+2. **Student Detail.** This view contains an expanded contact section containing siblings and other guardians. The student detail view also shows teachers how their students answered each survey.
+3. **Survey Results.** This allows teachers to look into a survey and see how their students responded. The survey viewer provides a quick summary of the results and an easy navigation to the individual student’s responses.
+
+Project Buzz also provides a way for teachers to add notes/feedback on their students. This functionality provides an important way for teachers to include commentary and updates in conjunction with the contact information and student responses they are already working with.
+
+> [!INFO]
+> View a 3 minute overview of the Buzz tool [here](https://vimeo.com/edfialliance/review/476016090/458295ac13).
```

Comment:
We'll want to fix this, but it isn't a dealbreaker. Can merge without changing the admonition. We should search out any other overlooked admonitions during the cleanup process. I'll put a note in the "post migration cleanup" ticket.